File: maxhanna.client/src/services/datacontracts/user/trophy.ts
import { FileEntry } from "../file/file-entry";

export class Trophy {
  id: number;
  name?: string;
  file?: FileEntry;

  constructor(id: number, name?: string, file?: FileEntry) {
    this.id = id;
    this.name = name;
    this.file = file;
  }
}

File: maxhanna.Server/Controllers/DataContracts/Users/Trophy.cs
using maxhanna.Server.Controllers.DataContracts.Files;

namespace maxhanna.Server.Controllers.DataContracts.Users
{
	public class Trophy
	{
		public int Id { get; set; }
		public string? Name { get; set; }
		public FileEntry? File { get; set; }
	}
}
